PURL-SPECIFICATION.rst

```diff
@@ -114,9 +114,11 @@ Rules for each ``purl`` component
 
 A ``purl`` string is an ASCII URL string composed of seven components.
 
-Some components are allowed to use other characters beyond ASCII: these
-components must then be UTF-8-encoded strings and percent-encoded as defined in
-the "Character encoding" section.
+Except as expressly stated otherwise in this section, each component:
+
+- MAY be composed of any of the characters defined as "Permitted Characters" in
+  the "Character encoding" section
```

Comment:
Here we are talking about the **serialized** form of a component (e.g. a package with name `Pan%E8%AA%9E`).

I would rather use "Permitted Bytes" instead of "Permitted Characters" to stress out that we are in the codomain of the `percent-encode` function.

Reply:
@ppkarwasz Thanks, but let's not introduce bytes just yet.